packages/jsonapi/node-tests/middleware-test.js

```diff
@@ -99,17 +102,17 @@ describe('jsonapi/middleware', function() {
             })
           ]
         )
-        ).withRelated(
-          'article-links',
-          [
-            factory.addResource('article-links', 'link-1').withAttributes({
-              url: 'http://cardstack.com/cards/articles/1'
-            }),
-            factory.addResource('article-links', 'link-2').withAttributes({
-              url: 'http://cardstack.com/cards/articles/3'
-            })
-          ]
-        );
+      ).withRelated(
+        'article-links',
+        [
+          factory.addResource('article-links', 'link-1').withAttributes({
+            url: 'http://cardstack.com/cards/articles/1'
+          }),
+          factory.addResource('article-links', 'link-2').withAttributes({
+            url: 'http://cardstack.com/cards/articles/3'
+          })
+        ]
+      );
```

Comment:
Given what I said above about moving the cardstack-queries preprocessing down into hub/writers.js, the way we will handle them in JSONAPIFactory is just the literal way:

```
withRelated('articles-links', [factory.addResource('cardstack-queries')...
```

This would then have the benefit of working the same whether you're doing it on the client or in a node test (we run the same JSONAPIFactory in both places).